services/src/Application/Users/Queries/GetUser/GetUserQuery.cs

```diff
@@ -3,4 +3,4 @@
 
 namespace Application.Users.Queries.GetUser;
 
-public sealed record GetUserQuery(string Username) : IRequest<UserDto>;
\ No newline at end of file
+public sealed record GetUserQuery(string Username) : IRequest<UserDetailsDto>;
```

Comment:
Why did you change it? This query was intended to return simplified user information for other regular users to view